azure_img_utils/compute.py

```diff
@@ -57,6 +57,59 @@ def create_image(
     return image_name
 
 
+def create_gallery_image_def_version(
```

Comment:
`def` should be expanded to avoid the reader thinking about `default` when in this case `def` stands for `definition`